### PR TITLE
chore: add explicit reference for code of conduct in parent readme.md

### PR DESCRIPTION
**Pull request by avishnu**
The "Update and document" section in the Sandbx onboarding ticket (https://github.com/cncf/sandbox/issues/299) lists this as a mandatory requirement.

### CODE REVIEW

**Review by Abhinandan-Purkait — APPROVED** [review]

**Review by niladrih — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -135,3 +135,5 @@ See [Vision](https://github.com/openebs/community/blob/develop/VISION.md) for mo
 ## New Contributors and Maintainers
 We are actively looking for new contributors and maintainers. Want to get involved, but not sure how? Please email the  maintainers at openebs.team@gmail.com
 
+## OpenEBS Community Code of Conduct
+OpenEBS follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```
Comment (tiagolobocastro) [inline profile/README.md:139]: Shouldn't it point to the community repo like we do here:
https://raw.githubusercontent.com/openebs/openebs/refs/heads/main/CODE_OF_CONDUCT.md

Comment (avishnu) [reply profile/README.md:139]: Not here. It's expected to explicitly point to the CNCF code of conduct. I've checked on other projects like Kusionstack.

Comment (tiagolobocastro) [reply profile/README.md:139]: Not sure why would it be different here, what do you mean by "it's expected"? Is this coming from a guideline?

Comment (avishnu) [reply profile/README.md:139]: `Ensure that that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub.`

This is from the onboarding ticket.

Comment (tiagolobocastro) [reply profile/README.md:139]: Not sure if that's what explicitly means here, I'll ask

Comment (tiagolobocastro) [reply profile/README.md:139]: I got no reply from CNCF, but seems most projects do this, so let's just follow suit.

**Review by tiagolobocastro — APPROVED** [review]

**Pull request merged**